web-ts/src/components/LocationHeader.tsx

```diff
@@ -45,7 +43,7 @@ const LocationHeader = (props: PropsType): ReactElement => {
   const eventsPath = createPath(EVENTS_ROUTE, params)
   const offersPath = createPath(OFFERS_ROUTE, params)
   const poisPath = createPath(POIS_ROUTE, params)
-  const newsPath = createPath(pushNotificationsEnabled ? LOCAL_NEWS_TYPE : TU_NEWS_TYPE, params)
+  const newsPath = createPath(pushNotificationsEnabled ? LOCAL_NEWS_ROUTE : TU_NEWS_ROUTE, params)
```

Comment:
is this the same as it was in the old web app? Looks a little bit unintuitive.

Reply:
Yes, its the same. If local news are not enabled (i.e. `pushNotificationsEnabled === false`) we link to tünews, otherwise to lcoal news.